invenio_oaiserver/query.py

```diff
@@ -59,10 +58,38 @@ def get_records(page=1):
 
     for result in response['hits']['hits']:
         yield {
-            # FIXME
             "id": result['_id'],
             "json": result['_source'],
-            # FIXME retrieve from elastic search
-            "updated": RecordMetadata.query.filter_by(
-                id=result['_id']).one().updated
+            "updated": datestamp_to_datetime(result['_source']['_updated'])
```

Comment:
Please keep the `RecordMetadata.query.filter_by(id=result['_id']).value('updated')` as default value in case the `_updated` key is not in `result['source']` (`result['_source'].get(..., RecordMetadata...)`).